scss/main.scss

```diff
@@ -806,6 +806,29 @@ h1,h2,h3 {
   margin: 0;
 }
 
+.iconContainer,
+.iconTile {
+	position: relative;
+	width: 100%;
+	min-height: 1px;
+	padding-right: 15px;
+	padding-left: 15px;
+}
+
+.iconContainer {
+	-webkit-box-flex: 0;
```

Comment:
Not necessary, but you can remove any properties with vendor prefixes (e.g. delete `-webkit-box-flex: 0;` but leave `flex: 0 0 83.33333%`)
The autoprefixer should take care of these.